zetaclient/chains/evm/observer/observer.go

```diff
@@ -95,7 +95,12 @@ func NewObserver(
 		evmJSONRPC:                    evmJSONRPC,
 		outboundConfirmedReceipts:     make(map[string]*ethtypes.Receipt),
 		outboundConfirmedTransactions: make(map[string]*ethtypes.Transaction),
-		priorityFeeConfig:             priorityFeeConfig{},
+		priorityFeeConfig: priorityFeeConfig{
```

Comment:
Just to confirm, to prevent: 
- 1 voting new priorityFee?
- 2 Using EIP1559 for the transactions?

If we have 1 but not 2, isn't some possible issue of using eip1559 while priorityFee is not updated?

Reply:
checked bool means we've already determined whether this chain supports priority fee. from the second time and on  i will directly use supported bool